src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -357,6 +358,14 @@ def test_stream_failed_run_logs(self, pf, capfd: pytest.CaptureFixture):
         out, _ = capfd.readouterr()
         assert "Input 'question' in line 0 is not provided for flow 'Simple_mock_answer'." in out
 
+    def test_failed_run_to_dict_exclude(self, pf):
+        failed_run = pf.runs.get(run="3dfd077a-f071-443e-9c4e-d41531710950")
```

Comment:
make the id a variable or something?

Reply:
will make this a variable for now, and will make it a new created run when migrate live test to canary.

Reply:
made a var